user-guide-chapters/server-configuration/security.adoc

```diff
@@ -0,0 +1,25 @@
+== {zotUpperName} Security and Hardening
```

Comment:
Should probably just call this hardening.

Reply:
There is also authorization:
https://github.com/project-zot/zot/blob/main/examples/config-policy.json